proto/scripts/swagger.py

```diff
@@ -36,12 +36,6 @@ def clean(path: str, patch: str) -> None:
     with open(path, "r") as f:
         spec = json.load(f)
 
-    # Update path names to be consistent.
-    paths = {}
-    for key, value in spec["paths"].items():
-        paths[key.replace(".", "_")] = value
-    spec["paths"] = paths
-
```

Comment:
so it was this + collisions?

Reply:
this added the collisions where `patch /abc/model.name` and `get /abc/model_name` collieded

Reply:
Why was this originally added, do you know?

Reply:
@hamidzr I'm still trying to understand the implications of this change.  I see a code diff in the generated files like this:

```diff
<     "/api/v1/checkpoints/{checkpoint.uuid}/metadata": {
---
>     "/api/v1/checkpoints/{checkpoint_uuid}/metadata": {
```

But I don't understand the significance.  Is that just purely cosmetic or does it affect the resulting API somehow?

Reply:
hmm my understanding is that in cases where it didn't used to cause collisions, this was purely visual. I can't say what each language plugin for swagger could do differently based on this change but the only binding we are actively using is working [fine](https://github.com/determined-ai/determined/pull/1437/checks?check_run_id=1249986030)

in general, I think if we don't have a good reason to modify this generated swagger spec, which is what this used to do,  we should avoid it as further demonstrated by the bug [here](https://determinedai.atlassian.net/browse/DET-4213)

Reply:
cool